base/subarray.jl

```diff
@@ -123,8 +123,10 @@ _maybe_reshape_parent(A::AbstractArray, ::NTuple{N, Bool}) where {N} = reshape(A
     view(A, inds...)
 
 Like [`getindex`](@ref), but returns a view into the parent array `A` with the
```

Comment:
```suggestion
Like [`getindex`](@ref), but returns a lightweight array that lazily references
(or is effectively a _view_ into) the parent array `A` at the given index or indices
`inds` instead of eagerly extracting elements or constructing a copied subset.
Calling [`getindex`](@ref) or [`setindex!`](@ref) on the returned value
(often a [`SubArray`](@ref)) computes the indices to access or modify the
parent array on the fly (without checking bounds a second time).
```